### ISSUE TITLE
[collector] So I tested the new helm chart version 0.97.2 with the `helm template` command and the result gives me a serviceMonitor which appears to have a wrong intend for the `relabelings` and `metricRelabelings`:

### ISSUE THREAD

**Issue by TylerHelmuth** [open-telemetry/opentelemetry-helm-charts#1287]
So I tested the new helm chart version 0.97.2 with the `helm template` command and the result gives me a serviceMonitor which appears to have a wrong intend for the `relabelings` and `metricRelabelings`:

```yaml
k apply -f - <<-EOT
apiVersion: monitoring.coreos.com/v1
kind: ServiceMonitor
metadata:
  name: otel
  namespace: default
  labels:
    helm.sh/chart: opentelemetry-collector-0.97.2
    app.kubernetes.io/name: otel-metrics
    app.kubernetes.io/instance: opentelemetry-collector
    app.kubernetes.io/version: "0.104.0"
    app.kubernetes.io/managed-by: Helm
spec:
  selector:
    matchLabels:
      app.kubernetes.io/name: otel-metrics
      app.kubernetes.io/instance: opentelemetry-collector
      component: standalone-collector
  endpoints:
  - port: metrics
  relabelings:
    - action: replace
      replacement: my-cluster
      targetLabel: cluster
  metricRelabelings:
    - action: keep
      regex: otelcol_process_memory_rss|target_info
      sourceLabels:
      - __name__
EOT
```

when I try to apply this manualy I get: `Error from server (BadRequest): error when creating "STDIN": ServiceMonitor in version "v1" cannot be handled as a ServiceMonitor: strict decoding error: unknown field "spec.metricRelabelings", unknown field "spec.relabelings"`

Am I the only one having this problem?

_Originally posted by @toporek3112 in https://github.com/open-telemetry/opentelemetry-helm-charts/issues/1240#issuecomment-2259391817_

**Comment by TylerHelmuth** [comment]
@rl0nergan please investigate

**Comment by TylerHelmuth** [comment]
I am not familiar with prometheus, but it looks to me like `relabelings` and `metricRelabelings` need to be part of [Endpoint](https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/api.md#monitoring.coreos.com/v1.Endpoint).

**Comment by rl0nergan** [comment]
Ah yep, that's my bad. Didn't indent enough. PRs for the fixes are below:
https://github.com/open-telemetry/opentelemetry-helm-charts/pull/1288
https://github.com/open-telemetry/opentelemetry-helm-charts/pull/1289

**Comment by jaronoff97** [comment]
closed by #1288 and #1289

**Comment by toporek3112** [comment]
Sorry for the late response, but I just wanted to say thank you for the support!
Everything is working perfectly now :)
